Assets/Editor/CreatureHydrator.cs

```diff
@@ -164,6 +164,14 @@ private static void HydrateCreature(UnitDefinition unit, string generaName, stri
             AnimatorStateTransition transitionToDieFromRun = runState.AddTransition(dieState);
             transitionToDieFromRun.AddCondition(AnimatorConditionMode.If, 0, "isDead");
         }
+        
+        if (dieState != null)
```

Comment:
lol makes me think of `dice` instead of `death`, maybe `deathState` or no since it breaks the convention?

Reply:
gonna leave for now because it would be a pretty big refactor